packages/legend-query/src/components/QueryBuilderPropertyExpressionEditor.tsx

```diff
@@ -166,7 +184,13 @@ const DerivedPropertyExpressionEditor = observer(
         <div className="panel__content__form__section__header__label">
           {derivedPropertyExpressionState.title}
         </div>
-        {!parameterValues.length && (
+        {(isDefaultDatePropagationSupported(
+          derivedPropertyExpressionState,
+          derivedPropertyExpressionState.queryBuilderState.graphManagerState
+            .graph,
+        )
+          ? !derivedPropertyExpressionState.parameters.length
+          : !parameterValues.length) && (
```

Comment:
Doesn't look right here, if we already filled the slots with `propagated values`, we should only care about `parameterValues.length`